### PR TITLE
removed remoteIP as a required parameter

### PR DESCRIPTION
**Pull request by wcalebgray**
as it is not required according to reCAPTCHA docs: https://developers.google.com/recaptcha/docs/verify

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,46 +12,45 @@ var options = {
 
 module.exports = function(privateKey, remoteIP, response, cb) {
   var error = null;
-  
+
   if (!privateKey) {
     error = 'Private key is required';
-  } else if (!remoteIP) {
```
Comment (wzrdtales) [inline index.js:18]: Just removing the conditions is not enough in this case. If remoteIP is optional now, we should make it also omitable in the call itself.

Comment (wcalebgray) [inline index.js:18]: The only way I can think to do this is to wrap the key, ip and response into a params object and pass that in along with the callback (e.g. function(params, cb) {}). This seems slightly more complicated than just telling the user to pass in 'undefined' for the second parameter. I'm open to other suggestions though. The IP is optional: https://developers.google.com/recaptcha/docs/verify

Comment (wzrdtales) [inline index.js:18]: Why so complicated?

``` javascript
function(privateKey, remoteIP, response, cb) {
  if( arguments.length < 4 && typeof(response) === 'function') {
    cb = response;
    response = remoteIP;
    remoteIP = undefined;
  }
}
```

Comment (wzrdtales) [inline index.js:18]: And changing the params in that way to expect an object would be a breaking change, thus a no go in this case.

**Comment by wcalebgray** [comment]
Updated to allow omission of ip as parameter into function.

File: index.js (hunk 1 above)
Comment (wcalebgray) [inline index.js:18]: Understood, good idea! Updated the PR.

**Comment by wzrdtales** [comment]
Thank you for your contribution!

**Pull request merged**